### ISSUE TITLE
set mode not displaying modes

### ISSUE THREAD

**Issue by thales-man** [fblackburn1/node-red-contrib-hubitat#159]
allow modes to be set is enabled in maker api.
the Hubitat config is correctly configured and fully operational.

the set mode node does not display the available modes in drop down.

device Hubitat c-8 pro patched to date (2.3.8.134)
node version 3.0.2
contrib version 1.9

**Comment by fblackburn1** [comment]
Hi @thales-man 
I cannot reproduce (but I don't have c-8 pro)


> the Hubitat config is correctly configured and fully operational.

Can you confirm that device node work as expected? (and dropdown too)

What is the behavior of the dropdown when you click on it? Only few choice? no choice at all? only the `Choose mode ...` ?
Do you have node-red logs when you click on the dropdown: [see documentation to enable debug](https://nodered.org/docs/user-guide/runtime/logging)

**Comment by thales-man** [comment]
Hi Francois, thank you for looking into this. I'd forgotten about it to be honest. I'll close it now.

As drop down wasn't populating, I'd resorted to using an HTTP call through to the hub to set them. Now, as you'd commented on it, I went back and added a 'set mode' node and the list in the drop down is now populating. Configuration is still the same, the only thing I have done is add a 'morning' mode. So I've removed the HTTP calls and inserted the 'set mode' nodes. Here they are now in all their glory:

<img width="728" alt="Screenshot 2024-05-16 at 12 44 22 PM" src="https://github.com/fblackburn1/node-red-contrib-hubitat/assets/17850586/e268bfb8-c244-4ca8-a8a3-77b00d43a135">
